### ISSUE TITLE
Everytime that the plugin updates, I lose my configs

### ISSUE THREAD

**Issue by Sponchiatto** [javalent/fantasy-statblocks#385]
### Scope

I use statblocks for a diferent system of RPG, I changed the layout to match my custom monsters. But everytime that the plugin updates I have to get my layout backup, import, then go to my creatures, save one by one again. I don't know if its a config or if the update overwrite everthing.

### Discovery

Maintain user settings

### Validate

I tested in obsidian, and I use Microsoft One Drive Cloud to store my vault backup, maybe this can influence in something.

**Comment by valentine195** [comment]
This only happens when you update the plugin? It doesn't happen if you close and reopen obsidian?

**Comment by Sponchiatto** [comment]
Yes, because when i went to configurations, community plugins menu, and clicked in check for updates and updated all plugins (fantasy statblock included), the problem of happened

It doesn't happen when I close and reopen obsidian, tried this right now.

**Comment by Sponchiatto** [comment]
@valentine195 It is a config from obsidian or the plugin?

**Comment by valentine195** [comment]
The plugin, under the “Advanced” section.

**Comment by Sponchiatto** [comment]
@valentine195 under "Advenced Settings"  section there is the option: "Try to Save Data Automatically" that is cheked. And next is layout, import homebrew creatures and Bestiary. I didn't find Atomic Write option.

**Comment by valentine195** [comment]
That is the setting I was talking about. Can you please try disabling it? I think that might be what is causing it.

**Comment by Sponchiatto** [comment]
Okay, I unchecked this option.
